io-package.json

```diff
@@ -202,7 +202,8 @@
         "adapterType": "zstack",
         "debugHerdsman": false,
         "disablePing": false,
-        "external": ""
+        "external": "",
+        "startWithInconsistent": false
```

Comment:
is this making sense to default to false?

Reply:
yes, this setting will only be used for critical cases

Reply:
https://github.com/Koenkk/zigbee-herdsman/issues/376

Reply:
is there an info which FWs are "old" and which not and where can I see the FW version? I have a stick since 3 years ... worked like a charm ... Honestly no idea which FW is on it

Reply:
![image](https://user-images.githubusercontent.com/8360230/128554597-d4821bc1-6952-435d-8417-9674728471fe.png)

for some cc2538 sticks with firmware before 10.2020
symptoms - a set of error messages on adapter start:
![image](https://user-images.githubusercontent.com/8360230/128554469-0c1538d3-e0d6-4db4-8ae6-d2efbd380f4f.png)

Reply:
the current solution is a compromise with those who cannot update the firmware or stick

Reply:
ok ... then lets see if I might be affected later ;-)))

type:zStack12
version:2-0.2.6.3.
revision:20180815

Reply:
the reason is more stringent requirements for the initialization-backup procedure in zigbee-herdsman

Reply:
these errors started in 1.5.6, if you didn't have them - everything is fine, you don't need this option

Reply:
now we will check on some users and release 1.6.0